test/core/util/fake_stats_plugin.cc

```diff
@@ -0,0 +1,85 @@
+// Copyright 2023 The gRPC Authors.
```

Comment:
Is this supposed to be 2024 now?

Reply:
We started writing this code in 2023, so I think that's the right year to use.